doc/api/esm.md

```diff
@@ -309,6 +309,35 @@ modules it can be used to load ES modules.
 The `import.meta` meta property is an `Object` that contains the following
 properties.
 
+### `import.meta.dirname`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.2 - Release candidate
+
+* {string} The directory name of the current module. This is the same as the
+  [`path.dirname()`][] of the [`import.meta.filename`][].
+
+> **Caveat** only file-based modules support this property.
```

Comment:
Unless I'm missing something, this is straight up false, a virtual module would have this property as long as its URL is under the `file` scheme.

```suggestion
> **Caveat**: only present on `file:` modules.
```